<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background:Gainsboro;
}
QPushButton{
background:Silver;
}
QPushButton:hover{
background-color:DarkGrey;
}
QPushButton:pressed{
background-color:Grey;
}</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="digitCount">
    <number>10</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="equal">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>207</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="division">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>207</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ac">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>75</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="degree">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>75</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="three">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>174</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="one">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>174</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="two">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>174</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="point">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>207</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="umn">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>174</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zero">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>207</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fact">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>75</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seven">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>108</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eight">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>108</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sqrt">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>75</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>√</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minus">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>141</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="five">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>141</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="six">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>141</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plus">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>108</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="four">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>141</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nine">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>108</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
